compat/src/index.js

```diff
@@ -186,8 +186,9 @@ let Children = {
 	},
 	only(children) {
 		children = toChildArray(children);
-		if (children.length !== 1)
+		if (children.length !== 1) {
```

Comment:
wait doesn't the git pre-commit hook not revert this change for you?

Reply:
Seems to keep it if it's there